<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OSCAR - Cluster Management Software</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QStackedWidget" name="stackedOptionsWidget" >
    <property name="geometry" >
     <rect>
      <x>190</x>
      <y>10</y>
      <width>1021</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex" >
     <number>0</number>
    </property>
    <widget class="QWidget" name="page" >
     <widget class="QTabWidget" name="ConfigurationWidget" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1011</width>
        <height>701</height>
       </rect>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab" >
       <attribute name="title" >
        <string>Distribution Configuration</string>
       </attribute>
       <widget class="QWidget" name="verticalLayout_3" >
        <property name="geometry" >
         <rect>
          <x>490</x>
          <y>60</y>
          <width>160</width>
          <height>149</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="refreshListSetupDistrosButton" >
           <property name="text" >
            <string>R&amp;efresh</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addDistroButton" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeDistroButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="advancedDistroButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>A&amp;dvanced</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_5" >
        <property name="geometry" >
         <rect>
          <x>20</x>
          <y>10</y>
          <width>191</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text" >
         <string>List of Linux Distributions That Are Setup:</string>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QListWidget" name="listSetupDistrosWidget" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>60</y>
          <width>461</width>
          <height>601</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2" >
       <attribute name="title" >
        <string>OSCAR Repositories</string>
       </attribute>
       <widget class="QWidget" name="verticalLayout_2" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>50</y>
          <width>190</width>
          <height>211</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="AddOSCARRepoButton" >
           <property name="text" >
            <string>Add OSCAR Repository</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Save Repository Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mirrorButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Mirror Repository</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Delete Repository</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="refreshListOPKGsButton" >
           <property name="text" >
            <string>Refresh List OPKGs</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayout_2" >
        <property name="geometry" >
         <rect>
          <x>210</x>
          <y>50</y>
          <width>791</width>
          <height>601</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QListWidget" name="listReposWidget" />
         </item>
         <item>
          <widget class="QListWidget" name="listOPKGsWidget" />
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_2" >
        <property name="geometry" >
         <rect>
          <x>620</x>
          <y>30</y>
          <width>381</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>Available OSCAR Packages in Selected Repositories</string>
        </property>
       </widget>
       <widget class="QLabel" name="label" >
        <property name="geometry" >
         <rect>
          <x>220</x>
          <y>30</y>
          <width>361</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text" >
         <string>Available OSCAR Repositories</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3" >
       <attribute name="title" >
        <string>Sanity Check</string>
       </attribute>
       <widget class="QWidget" name="verticalLayout_4" >
        <property name="geometry" >
         <rect>
          <x>10</x>
          <y>20</y>
          <width>221</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="systemSanityCheckButton" >
           <property name="text" >
            <string>System Sanity Check</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="oscarSanityCheckButton" >
           <property name="text" >
            <string>OSCAR Sanity Check</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="saveSanityCheckLogsButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="geometry" >
         <rect>
          <x>11</x>
          <y>630</y>
          <width>219</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text" >
         <string>Save Logs...</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="sanityCheckTextWidget" >
        <property name="geometry" >
         <rect>
          <x>235</x>
          <y>11</y>
          <width>761</width>
          <height>651</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3" >
     <widget class="QTabWidget" name="networkConfigurationTabWidget" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1021</width>
        <height>701</height>
       </rect>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_4" >
       <attribute name="title" >
        <string>Page</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayout" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>720</y>
      <width>1181</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="QuitButton" >
       <property name="text" >
        <string>Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listOscarOptionsWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>701</height>
     </rect>
    </property>
    <property name="lineWidth" >
     <number>3</number>
    </property>
    <property name="iconSize" >
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="spacing" >
     <number>10</number>
    </property>
    <property name="viewMode" >
     <enum>QListView::IconMode</enum>
    </property>
    <item>
     <property name="text" >
      <string>OSCAR Configuration</string>
     </property>
     <property name="icon" >
      <iconset resource="../xoscar_resource.qrc" >:/new/prefix1/icons/configure.png</iconset>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Cluster Management</string>
     </property>
     <property name="icon" >
      <iconset resource="../xoscar_resource.qrc" >:/new/prefix1/icons/kcontrol.png</iconset>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1214</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionOSCAR_Related_Projects" />
    <addaction name="actionAbout_OSCAR" />
    <addaction name="actionAboutXOSCAR" />
   </widget>
   <addaction name="menuAbout" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <number>4</number>
   </attribute>
  </widget>
  <action name="actionAboutXOSCAR" >
   <property name="text" >
    <string>About xOSCAR</string>
   </property>
  </action>
  <action name="actionAbout_OSCAR" >
   <property name="text" >
    <string>About OSCAR</string>
   </property>
  </action>
  <action name="actionOSCAR_Related_Projects" >
   <property name="text" >
    <string>OSCAR Related Projects</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../xoscar_resource.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>QuitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>715</x>
     <y>770</y>
    </hint>
    <hint type="destinationlabel" >
     <x>715</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
